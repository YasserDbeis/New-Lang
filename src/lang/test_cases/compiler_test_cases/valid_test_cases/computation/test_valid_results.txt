func main()
{
    print(2 + 2 * 3 - 4 / 2 + 1);
    print((2 + 2) * 3 - 4 / (2 + 1));
    print(2 + 2 * 3 - 4 / 2 + 1);
    print(2.0 * 3.0);
    print(2.0 + 1);
    print(1 / 2.0);
    print("test" + 123);
    print("test" + 123.0);
    print("test" + true);
    print("test" + false);
    print("test" + "test");
    print(2 is 2);
    print(2 != 2);
    print(2 < 3);
    print((2 is 2) and (2 != 2));
    print((2 != 2) or (2 < 3));
    print((2 < 3) != ((2 is 2) and (2 != 2)));
}