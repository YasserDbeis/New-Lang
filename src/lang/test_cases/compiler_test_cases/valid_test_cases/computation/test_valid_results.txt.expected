7.000000
11.000000
7.000000
6.000000
3.000000
0.500000
test123
test123.0
testtrue
testfalse
testtest
true
false
true
false
true
true